### PR TITLE
modifed readme, added community section.

### PR DESCRIPTION
**Pull request by Shardul555**

### CODE REVIEW

**Comment by bymayanksingh** [comment]
@Shardul555 let's be more descriptive with pull requests from next time, it's important that the maintainer knows what does your PR have to offer.

File: README.md [hunk 1]
```diff
@@ -156,6 +159,27 @@ For more details and available options, please check their [timothycrosley/isort
 
 Close issues using keywords: [how to ?](https://help.github.com/en/articles/closing-issues-using-keywords)
 
+## :cherry_blossom: Community
+___
+
+If it's open-source, talk about the community here, ask social media links and other links.
```
Comment (bymayanksingh) [inline README.md]: please remove this line.

**Review by bymayanksingh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bymayanksingh) [reply README.md]: line 165 specifically.

**Comment by bymayanksingh** [comment]
fixes for #26

**Comment by bymayanksingh** [comment]
I am doing the necessary changes, thanks for your PR. :)

**Review by bymayanksingh — APPROVED** [review]

**Comment by Shardul555** [comment]
Thanks sir for giving me this opportunity to learn.

**Pull request merged**